src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -745,18 +760,30 @@ private CachedCase createNewCase(Long uprn) throws CTPException {
     // Query AIMS for UPRN
     AddressIndexAddressCompositeDTO address = addressSvc.uprnQuery(uprn);
 
-    if (address.getCountryCode() == SCOTTISH_COUNTRY_CODE) {
+    if (SCOTTISH_COUNTRY_CODE.equals(address.getCountryCode())) {
```

Comment:
Scottish is the language...

Reply:
Scottish, relating to Scotland? Changed to SCOTLAND_COUNTRY_CODE to avoid ambiguity.